README.md

```diff
@@ -1,10 +1,39 @@
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/a5ecnenoai4ydkrv/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/lifecycleassessment_toolkit/branch/master) [![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/LifeCycleAssessment_Toolkit/LifeCycleAssessment_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=52&branchName=master)
```

Comment:
Definition ID for Azure should be `193` - full badge should be `[![Build Status](https://dev.azure.com/BHoMBot/BHoM/_apis/build/status/LifeCycleAssessment_Toolkit/LifeCycleAssessment_Toolkit.CheckCore?branchName=master)](https://dev.azure.com/BHoMBot/BHoM/_build/latest?definitionId=193&branchName=master)`